pkg/apply/quickstart.go

```diff
@@ -37,6 +36,7 @@ func Quickstart(clientFactory factory.Factory, version, fqdn string) error {
 
 	// TODO: We should obtain settings like these from a config that can be ingested at runtime.
 	// Storing these here for now until we have a solution
+	// https://github.com/ForgeRock/forgeops-cli/issues/58
```

Comment:
Created a new issue to track cli configs. We shouldn't hardcode this here.
see #58